modules/cli/src/main/scala/scala/cli/commands/Compile.scala

```diff
@@ -10,6 +10,7 @@ object Compile extends ScalaCommand[CompileOptions] {
   override def group                                  = "Main"
   override def sharedOptions(options: CompileOptions) = Some(options.shared)
   def run(options: CompileOptions, args: RemainingArgs): Unit = {
+    options.helpGroups.printHelp(RunOptions.help)
```

Comment:
If these options are pulled via `SharedOptions` (see my comment below), maybe a helper method could be added to `ScalaCommand`, like
```scala
def maybePrintGroupHelp(options: T): Unit =
  for (shared <- sharedOptions(options)) {
    shared.helpGroups.maybePrintGroupHelp(help)
  }
```
(`ScalaCommand` already has the `Help[_]` instance, and has a method to extract the shared options from the sub-command ones.)

Then, sub-commands would just have to call
```scala
maybePrintGroupHelp(options)
```

Reply:
Good point, thanks for review.